docs/reference/monitoring/index.asciidoc

```diff
@@ -1,54 +1,39 @@
 [role="xpack"]
 [testenv="basic"]
-[[es-monitoring]]
-= Monitoring {es}
+[[monitor-cluster]]
```

Comment:
Since this is a top-level heading, I'd be inclined to be redundant and include "elasticsearch" in the anchor: [monitor-elasticsearch-cluster]. Similarly, I'd insert {es} before _cluster_ in the paragraph below.